linker/shared/src/main/scala/org/scalajs/linker/checker/IRChecker.scala

```diff
@@ -164,6 +165,37 @@ private final class IRChecker(unit: LinkingUnit, logger: Logger) {
     }
   }
 
+  private def checkDuplicateMembers(classDef: LinkedClass): Unit = {
+    // Fields
+    val declaredFields = mutable.Set.empty[(Boolean, FieldName)]
+    for {
+      field @ FieldDef(flags, FieldIdent(name, _), _) <- classDef.fields
+    } {
+      implicit val ctx = ErrorContext(field)
```

Comment:
I'm not sure this is the right `ErrorContext`. The field definition on its own is fine, it's the class def that is broken, no?